http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala

```diff
@@ -455,11 +536,28 @@ class AzurePubsubHandlerInterp[F[_]: Parallel](
       resourceId
     )
 
+  private def getCommonFieldsForWsmGeneratedClient(name: ControlledResourceName,
+                                                   resourceDesc: String,
+                                                   userEmail: WorkbenchEmail
+  ) =
+    new ControlledResourceCommonFields()
+      .accessScope(bio.terra.workspace.model.AccessScope.PRIVATE_ACCESS)
```

Comment:
Quick question, would the accessScope be different for creating apps using pre-existing databases in cloned workspace? Definitely outside of the scope of this PR, but something to be careful with once we get to polling for app resources creation @lucymcnatt